packages/fxa-settings/package.json

```diff
@@ -71,6 +71,7 @@
     "eslint-plugin-jest": "^23.13.2",
     "eslint-plugin-react": "7.19.0",
     "fxa-shared": "workspace:*",
+    "jest-mock-extended": "^1.0.10",
```

Comment:
I'm definitely not sure this is actually needed, it just made mocking out the `ErrorResponse` arguments easier. Happy to use something else to keep our dependencies down 🤷

Reply:
in this case since the mocks aren't being used/checked we should be able to just do this in the test file to satisfy the compiler:
```ts
const operation = null as any as Operation;
const forward = null as any as NextLink;
```